scripts/prepare-release.js

```diff
@@ -117,20 +175,45 @@ async function updatePatchVersions() {
       });
     }
   }
+}
 
-  await fs.writeJSON(
-    patchedJsonPath,
-    { currentReleaseVersion: {} },
-    { spaces: 2, encoding: 'utf8' },
-  );
+/**
+ * Detects any patched packages version since the most recent release on
+ * the main branch, and then bumps all packages in the repo accordingly.
+ */
+async function updatePackageVersions(repo) {
+  const rootPkgPath = path.resolve(repo.root.dir, 'package.json');
+  const { version: currentRelease } = await fs.readJson(rootPkgPath);
```

Comment:
now there is, realized that looking up the previous release in git is silly, as it's already here xP